etc/adminhtml/system.xml

```diff
@@ -69,6 +82,7 @@
                 <field id="email" translate="label comment" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Email</label>
                 </field>
+                <comment>NOTE: Support for Klaviyo user authentication is set to deprecate.  We advise setting up access via the Setup Oauth tab</comment>
```

Comment:
I think we can get rid of this or change the description here since we won't be using the klaviyo user tabe?

Reply:
yes I am planning on removing it, the latest push was just to rebase with master